projects/luajit/README.md

```diff
@@ -0,0 +1,82 @@
+# LuaJIT
+
+[LuaJIT][luajit-homepage] is a Just-In-Time Compiler (JIT) for the Lua
+programming language. Lua is a powerful, dynamic and light-weight programming
+language. It may be embedded or used as a general-purpose, stand-alone
+language.
+
+## Build Docker
+
+    $ sudo docker build -t oss-sydr-fuzz-luajit .
+
+## Run Hybrid Fuzzing
+
+Unzip Sydr (`sydr.zip`) in `projects/luajit` directory:
+
+    $ unzip sydr.zip
+
+Run docker:
+
+    $ sudo docker run --privileged --network host -v /etc/localtime:/etc/localtime:ro --rm -it -v $PWD:/fuzz oss-sydr-fuzz-luajit /bin/bash
+
+Change directory to `/fuzz`:
+
+    # cd /fuzz
+
+Run hybrid fuzzing:
+
+    # sydr-fuzz -c luaL_loadstring.toml run
+
+Corpus minimization (step is required for AFL++):
+
+	sydr-fuzz -c luaL_loadstring.toml cmin
+
+Collect and report coverage:
+
+    # sydr-fuzz -c luaL_loadstring.toml cov-report
+
+Building HTML report:
+
+	$ sydr-fuzz -c luaL_loadstring.toml cov-show -- -format=html > index.html
```

Comment:
Change TAB to spaces and change `$` to `#`.

Reply:
Fixed here and above.